main/universal/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/wal-g/wal-g/cmd/etcd"
+	"github.com/wal-g/wal-g/cmd/fdb"
+	"github.com/wal-g/wal-g/cmd/gp"
+	"github.com/wal-g/wal-g/cmd/mongo"
+	"github.com/wal-g/wal-g/cmd/mysql"
+	"github.com/wal-g/wal-g/cmd/pg"
+	"github.com/wal-g/wal-g/cmd/redis"
+	"github.com/wal-g/wal-g/cmd/sqlserver"
+)
+
+var (
+	universalCmd = &cobra.Command{
+		Use:   "wal-g",
+		Short: "Universal database backup tool",
+	}
+)
+
+func main() {
+	etcdCmd := etcd.GetCmd()
+	etcdCmd.Use = "etcd"
+	universalCmd.AddCommand(etcdCmd)
+
+	fdbCmd := fdb.GetCmd()
+	fdbCmd.Use = "fdb"
+	universalCmd.AddCommand(fdbCmd)
+
+	gpCmd := gp.GetCmd()
```

Comment:
gpCmd reuses postgres for segments https://github.com/wal-g/wal-g/blob/5d661e588c8df623fdf9448290c58176f4ede420/cmd/gp/gp.go#L58  Seems it will affect universal command.